p11-kit/p11-kit.c

```diff
@@ -49,6 +49,14 @@
 #include <stdlib.h>
 #include <unistd.h>
 
+#ifdef ENABLE_NLS
+#include <libintl.h>
+#define _(x) dgettext(PACKAGE_NAME, x)
+#else
+#define _(x) (x)
+#endif
+#define N_(x) (x)
```

Comment:
what's this for?

Reply:
It's a marker for the xgettext program to extract the translatable strings into POT files, but not replace the text at run time:
https://developer.gnome.org/glib/stable/glib-I18N.html#N-:CAPS